packages/ses/src/error/tame-v8-error-constructor.js

```diff
@@ -285,7 +287,7 @@ export const tameV8ErrorConstructor = (
 
   const defaultPrepareFn = tamedMethods.prepareStackTrace;
 
-  OriginalError.prepareStackTrace = defaultPrepareFn;
+  const originalPrepareStackTrace = OriginalError.prepareStackTrace;
```

Comment:
The old code attenuated the `OriginalError.prepareStackTrace`. The new code does not. But `OriginalError` *should* be encapsulated within this taming. The powerful dangerous error exposed on the start compartment is `InitialError`, whose taming should equivalently attenuate its `InitialError.prepareStackTrace` access property. Yes?

Additionally, the old code made no use of what happened to already be the value of `OriginalError.prepareStackTrace` before SES started. Instead, the taming implicitly assumed either that it would be as v8 left it. If the host had modified it before SES started (as it turns out Node does), that we would ignore that replacement behavior. Yes?

This PR captures this original `prepareStackTrace` function, whatever it is, and then initializes the `InitialError.prepareStackTrace` accessor property by reusing the attenuation it already implements in its setter. Yes?

Reply:
The new code does attenuate the original `OriginalError.prepareStack` when it uses it with the `InitialError.prepareStack` setter below. I can add a comment to this effect to make this more clear.

Indeed the original code completely discarded the original `prepareStack`. It's a stated goal of this PR to no longer do so as the host, in particular Node.js, may have set it up to something that helps handle stack traces.

Reply:
> I can add a comment to this effect to make this more clear.

Yes please. Resolving.